modules/marketplace/component/CategoriesNav/ButtonCat.tsx

```diff
@@ -49,12 +51,12 @@ const ButtonCat = ({ category, index, handleActiveNav, active }: CategoriesProps
         onMouseOver={() => setPopupClass(true)}
         onMouseLeave={() => setPopupClass(false)}
       >
-        {category.subcategories.map((item: { name: string }, i: number) => (
```

Comment:
why did you remove the typing?

Reply:
the typing was imporoved when hovered on the type is
``` javascipt
(parameter) item: {
    name?: string | undefined;
}
```typescript
This is the real type, it was removed because it was not neccessary again after the refactoring

export type CategoryType = {
  name: string;
  subcategories: {name?: string}[];
};
``